homework_14/homework_14.py

```diff
@@ -0,0 +1,28 @@
+class Student:
+    def __init__(self, first_name, last_name, age, average_mark):
```

Comment:
It is better to create this properties (name, age, etc) as fields at the start of class.